src/backend/metal/src/command.rs

```diff
@@ -1963,14 +1963,78 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         T: IntoIterator,
         T::Item: Borrow<com::BufferCopy>,
     {
-        let commands = regions.into_iter().map(|region| {
-            soft::BlitCommand::CopyBuffer {
-                src: src.raw.clone(),
-                dst: dst.raw.clone(),
-                region: region.borrow().clone(),
+        let inner = unsafe { &mut *self.inner.get() };
+        let compute_pipe = self.shared.service_pipes
+            .lock()
+            .unwrap()
+            .get_copy_buffer()
+            .to_owned();
+
+        let mut blit_commands = Vec::new();
+        let mut compute_commands = vec![
+            soft::ComputeCommand::BindPipeline(compute_pipe),
+        ];
+        let wg_size = MTLSize {
+            width: 64,
```

Comment:
Could just use `self.shared.device.max_threads_per_threadgroup().width` for now I guess

Reply:
We should get the optimal width from the [PSO itself](https://developer.apple.com/documentation/metal/mtlcomputepipelinestate/1414911-threadexecutionwidth?language=objc), but that's not critical